channeldb/channel.go

```diff
@@ -2353,7 +2353,7 @@ func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg,
 
 	var newRemoteCommit *ChannelCommitment
 
-	err := kvdb.Update(c.Db, func(tx kvdb.RwTx) error {
+	err := kvdb.Update(c.Db.chanCache, func(tx kvdb.RwTx) error {
```

Comment:
I don't think we want to cache this bucket (or rather the operations in this DB closure) since we end up accessing an ever-growing bucket (all the past revoked states). On larger nodes this may be in the GBs range (would need to use the bolt browser on a volunteer to get a better picture of things). 

With the current design, is it correct that we _must_ use the cache here, since otherwise we'd have consistency issues since other buckets like the revocation state get modified here?

Reply:
Okay this is somehting I didn't see properly, but yes you're correct if some of these channel state buckets grow indefinitely then we need to do something about evicting on the fly. Yes for now we need things together for consistency.